alien-module/prototypes/recipe/alien-warfare.lua

```diff
@@ -22,7 +22,7 @@ data:extend({
 		result = "alien-ore-magazine",
 		result_count = 1,
 		ingredients = {
-			{ "artifact-ore", 2 },
+			{ "artifact-ore", 1 },
```

Comment:
please revert

Reply:
The alien-ore-magazine (4 damage) is too expensive for this short-term use in the early game.